x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/mapper/SemanticTextFieldMapper.java

```diff
@@ -83,6 +85,8 @@ public class SemanticTextFieldMapper extends FieldMapper implements InferenceFie
     public static final NodeFeature SEMANTIC_TEXT_SEARCH_INFERENCE_ID = new NodeFeature("semantic_text.search_inference_id");
 
     public static final String CONTENT_TYPE = "semantic_text";
+    public static final String DEFAULT_INFERENCE_ID = ".elser_default_ingest";
+    public static final String DEFAULT_SEARCH_INFERENCE_ID = ".elser_default_search";
```

Comment:
These values are placeholders subject to change once we have the real default names